src/Polysemy/Lift.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Polysemy.Lift
+  ( -- * Effect
+    Lift (..)
+
+    -- * Actions
+  , lift
```

Comment:
Traditionally this has been called `sendM` but admittedly that is a crap name. I'm a little concerned about the overlap with `MonadTrans.trans` --- but I have to say this is nice. Thoughts?

Reply:
If we go this way, let's remove `sendM` across the codebase and reexport `lift` from `Polysemy`.

Reply:
Ah, I just did this because I was following convention (and was sleepy).

I like `lift` more, because there is no non-`Internal` `send` for `sendM` to be "based off". (like `map` -> `mapM`).

It has the same idea as `MonadTrans.lift` so I don't think that's an issue. (?)

Reply:
I went ahead and did the rename, if you want to use it.

Reply:
Thanks for the rename! Upon looking at the changes though, I'm already confused by the name. I think overloading `lift` here is only going to lead to confusion in the library. Can we come up with some other name (possibly also for the type itself?)

Reply:
I don't have any particularly bright ideas:

* `Name` - because you're giving a "name" (syntax) to a thing that already has an interpretation (inspired by logic, where you do the same when you want to refer to an element of your universe with a constant)

* `Concrete/Unambiguous` (or some nice sounding synonym) - because you're not leaving anything "open to interpretation" (in the Syntax vs Semantics sense in which the other "native" effects are open to "interpretation")